FULL_HELP_DOCS.md

```diff
@@ -1291,6 +1292,206 @@ Sign, Simulate, and Send transactions
 
 
 
+## `stellar tx add-op`
```

Comment:
I think we should make this command `stellar tx op add`. The command will grow a little more gracefully if we want to add remove op, update op, and typing hyphens on the terminal is annoying in commands and it's a graceful way to remove the hyphen.